<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlarmNetWindowClass</class>
 <widget class="QDialog" name="FlarmNetWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlarmNet database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SkTableView" name="flarmNetTable">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="toolTip">
        <string>Enter search text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearButton">
       <property name="toolTip">
        <string>Clear search text</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="createPlaneAction">
   <property name="text">
    <string>&amp;Create plane...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SkTableView</class>
   <extends>QTableView</extends>
   <header>src/gui/widgets/SkTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FlarmNetWindowClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
